pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/extensions/ExtensibleLoadManagerImpl.java

```diff
@@ -213,6 +214,19 @@ public static boolean debug(ServiceConfiguration config, Logger log) {
         return config.isLoadBalancerDebugModeEnabled() || log.isDebugEnabled();
     }
 
+    public static void createSystemTopic(PulsarService pulsar, String topic) throws PulsarServerException {
```

Comment:
Nit: why is this method `public` and `static`? It seems like it should be `private` since it is used only within an instance of this class.

Reply:
Oops, I missed that reference. I still worry about misuse of this method because it is public and static--it's probably not a large concern though.